tests/tasks/database/test_sqlite.py

```diff
@@ -0,0 +1,74 @@
+import os
+import pytest
+
+from prefect.tasks.database.sqlite import SQLiteQuery, SQLiteScript
+
+
+class TestSQLiteQuery:
+    def test_run_with_no_db(self):
+        sqlite_query = SQLiteQuery(query="SELECT * FROM table")
+
+        msg_match = "The db must be specified"
+        with pytest.raises(ValueError, match=msg_match):
+            sqlite_query.run()
+
+    def test_run_with_no_query(self):
+        sqlite_query = SQLiteQuery(db="database.db")
+
+        msg_match = "The query string must be specified"
+        with pytest.raises(ValueError, match=msg_match):
+            sqlite_query.run()
+
+    def test_run_with_no_param_data(self):
+        sqlite_query = SQLiteQuery(
+            db="database.db", query="SELECT * FROM table WHERE ?"
+        )
+
+        msg_match = "The data inputs must be specified as a tuple if query"
+        with pytest.raises(ValueError, match=msg_match):
+            sqlite_query.run()
+
+
+class TestSQLiteScript:
+    def test_run_with_no_db(self):
+        sqlite_script = SQLiteScript(script="SELECT * FROM table")
+
+        msg_match = "The db must be specified"
+        with pytest.raises(ValueError, match=msg_match):
+            sqlite_script.run()
+
+    def test_run_with_no_script(self):
+        sqlite_script = SQLiteScript(db="database.db")
+
+        msg_match = "The query script string must be specified"
+        with pytest.raises(ValueError, match=msg_match):
+            sqlite_script.run()
```

Comment:
Can you add a test that runs a script and then verify it works? Something simple like create a table and then query it.

Reply:
I think I did it in `test_sqlite_create_insert_select()`